### ISSUE TITLE
"Street life, saves lives" Aka Bike/PedSafetyMapping

### ISSUE THREAD

**Issue by layspmb** [atxhack4change/2016-project-proposals#14]
## Short description of the idea (140 character tweet):

Active streets with cyclists and pedestrians become safer streets. We need to share this knowledge with Austinites to strengthen biking and walking across the city.

## What is your project concept or idea? What challenge or opportunity will it solve?

The problem is that there is no place where cyclists and pedestrians can share and collect data about biking and pedestrian safety, use and infrastructure. This is an important community building element to encouraging more everyday cycling and walking because it over-comes perceived fear that many would be cyclists and pedestrians may have about using the city. It also opens up shared opportunities both for the user and for the City to learn from real-time user data.

The challenge is to create an app which collects tracking data for walking and cycling in the city and then gives the user an opportunity to qualitatively rate their experience for either a segment or the whole journey. Functionality to also add time coded comments about specific negatives/positives along the route would also be useful. The user would receive direction to the City’s 3-1-1 app if appropriate for loggin street infrastructure problems. 

The collection of this data would form a Stress Map of the city identifying most commonly used routes for either walking or biking and their relative quality as a color coding. Specific point comments would be clickable to pull up short comments by other users building the community feel.

### Do you have photos of them? Or quotes?

Primary – personal use (cyclist, pedestrians – tourists, residents)
Secondary – City planners, elected officials and their staff.

Primary: We heard from several UT students that they wanted to ride bikes in the city, but that their lack of knowledge of the city and their perceived feelings of lack of safety prevented this. They were worried that there was not a safe route to their destination. Google maps, identified bike lanes, but these vary dramatically in their quality and how safe they feel. This makes predicting a route for a novice or casual user a significant challenge as they may reach a point in their journey where they have to cycle or walk in a location they do not feel safe. This fear, result in them not adopting biking or walking as a mode of transit. This app would provide them with highly valuable crowd sourced information about the real qualitative nature of traversing the city by bike or on foot.

Secondary: The secondary users are the city staff who will be able to use the stress map output as a tool for deciding where capital investment needs to be made to improve or connect segments of the existing system. Understanding how people currently use the city and their qualitative view of the infrastructure provided will allow for better allocation of resources.

### [Look at the right sidebar :arrow_upper_right: ! Do you see the Labels? Pick a few]

- Education
- Health
- Mobility/Transportation
- Neighborhoods
- Public Safety
- Sustainability

## What is the current state of your project idea? [Pick more labels :arrow_upper_right: ]

- I have an idea or **concept** of a solution, but have not yet prototyped or tested it

### Do you have any links to text, mockups, code, or data?

The concept is based on several requisites. It needs to be app based, running on smart phones, capturing tracking data, and allowing for user pin drop comments. The City has been talking with Ride Report who produce an app which has some of this functionality specifically for cyclists. Therefore it is possible we could partner with Ride Report or build from their platform.

### Are you representing a group in submitting this project idea? 

I am an urban designer and planner working for Perkins + Will architects. I am chair of the Austin Pedestrian Advisory Council’s Technical Sub-committee and I also sit on the City’s Vision Zero Task Force. I therefore represent all of these groups, but most of all I am a user who wants to see increased street life in Austin. The bigger picture is that the more people who are comfortable using our streets for all forms of travel, be it walking, skateboarding, cycling, mobility impaired users, transit riders, pedi-cabs, and drivers, the better our streets will become at performing the vast number of task they need to achieve, as places we live, places we meet friends, places we people watch, places we eat out on, places we watch buskers and live music, and places we use to get where we are going.

**Comment by mateoclarke** [comment]
Thanks for submitting this project idea @layspmb.

There are two projects you might already be aware from other cities that might be helpful resources for this project:
- http://www.cyclephilly.org/ (this phone app has been forked and redeployed to other cities)
- http://project-pavement.herokuapp.com/

**Comment by ssharif1** [comment]
http://www.austinmonitor.com/stories/2016/05/city-ponders-giant-increase-sidewalk-investments/

Just read this!

**Comment by NicMoe** [comment]
Here's a map project looking at 'stress for cyclists' by road we might want to familiarize ourselves with: http://www.citylab.com/commute/2016/05/mapping-how-stressful-streets-can-be-for-cyclists/482469/

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by mateoclarke** [comment]
You're project has been accepted, @layspmb!

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email in the next few days asking you to confirming your speaking slot for the Saturday morning pitches. 

Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by mateoclarke** [comment]
@layspmb, I also saw that @Sahedeva pitched a similar project (but then closed it out) and I believe you both collaborated on the Code Across project planning. Just wanted to make sure everyone is all on the same page.

If you could, we still need your email address & name. You can comment back here, or email ssharif1@stedwards.edu with those details.

Thanks!

**Comment by Sahedeva** [comment]
Mateo,

I made my proposal b/c of the deadline and I didn’t see that the group already had a project set up. I was simply making sure we had a placeholder. Peter has been much more active with the project since CodeAcross so once I found out that he already had a project I naturally closed mine. I will be working on his project.

- Bob

On May 25, 2016, at 9:56 PM, Mateo Clarke <notifications@github.com<mailto:notifications@github.com>> wrote:


@layspmb<https://github.com/layspmb>, I also saw that @Sahedeva<https://github.com/Sahedeva> pitched a similar project (but then closed it out) and I believe you both collaborated on the Code Across project planning. Just wanted to make sure everyone is all on the same page.

If you could, we still need your email address & name. You can comment back here, or email ssharif1@stedwards.edu<mailto:ssharif1@stedwards.edu> with those details.

Thanks!

**Comment by layspmb** [comment]
Yep, we are on the same page. We are collaborating! Thanks, Peter

**Comment by jjoyce24** [comment]
Can I join the group? I went to school with @Sahedeva and would love to work on this project!

**Comment by layspmb** [comment]
@jjoyce24 Of course! very happy to have you involved. It should be a great Hack4Change.

**Comment by layspmb** [comment]
In case anyone needs them, my contact details are:
Peter Baird
layspmb@gmail.com

My project champion partners-in-crime are @NicMoe and Lauren :-)

**Comment by danielhonker** [comment]
@layspmb - you should have received an invite to join Slack (I think you are still on there from last year anyways). If you haven't already joined the team and seen the Slack channel for your project, it's #safety-mapping. We'll be sending out an email with details on pitches soon.

**Comment by Sahedeva** [comment]
The Hack-a-thon is coming up soon. Can anyone send an invite to the slack channel so we can coordinate - I tried searching for #safety-mapping but no results.
